.idea/gradle.xml

```diff
@@ -7,7 +7,6 @@
         <option name="testRunner" value="GRADLE" />
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleJvm" value="11" />
```

Comment:
Okay, seems you have some other settings inside android studio than I. Let's revert that please.